tests/Controllers/ItemsCsvImportTest.php

```
@@ -11,17 +11,16 @@
 use App\Models\Stock_location;
 use App\Models\Supplier;
-use Config\Database;
 
 class ItemsCsvImportTest extends CIUnitTestCase
 {
     use DatabaseTestTrait;
 
     protected $migrate = true;
     protected $migrateOnce = true;
-    protected $seed = '';
+    protected $seed = 'TestDatabaseBootstrapSeeder';
```

PHPStan (2.2.7)

[error] Property Tests\Controllers\ItemsCsvImportTest::$seed (''|class-string<CodeIgniter\Database\Seeder>|list<class-string<CodeIgniter\Database\Seeder>>) does not accept default value of type 'TestDatabaseBootstr…'.
Type `#3` from the union: 'TestDatabaseBootstr…' is not a list.

(property.defaultValue)